### PR TITLE
Docker & Docker compose

### PR DESCRIPTION
**Pull request by hec7orci7o**
## Use Docker to run the app
creation of a dockerfile that can be found in: `src/main/docker/app.dockerfile`
explanation can be found in `description.md` under the title of *`Docker Usage`*

## Use Docker Compose to run the app
creation of a docker-compose.yml that can be found in the `root directory`
explanation can be found in `description.md` under the title of *`Docker-compose`*

### CODE REVIEW

**Comment by fjlopez** [comment]
You have been awarded a :gift: for your contribution. However, you need to fix the minor issues found.

**Review by hec7orci7o — COMMENTED** [review]
I can't find the bugs you are talking about, do you mean checking the changes to be able to do the merge, if so, the only thing that is different is the readme which as you said I have added my status badge.

**Comment by fjlopez** [comment]
Do you cannot see the reviews?

**Comment by fjlopez** [comment]
Sorry, the reviews were unfinished. Now they are visible.

File: README.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ Some ideas for obtaining a :gift: if you are the first that:
 user name | NIA | GitHub Action |Score
 ----------|-----|---------------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml)
-your name | your NIA | your GitHub Action status
+[Hec7or-Uni](https://github.com/Hec7or-Uni/lab1-git-race) |798095 | [![Build Status](https://github.com/Hec7or-Uni/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/Hec7or-Uni/lab1-git-race/actions/workflows/ci.yml)
```
[the remark this answers is not in the document]
Comment (hec7orci7o) [reply README.md:17]: only 1?

[the remark this answers is not in the document]
Comment (fjlopez) [reply README.md:17]: Yep, only 1 per lab.

[the remark this answers is not in the document]
Comment (hec7orci7o) [reply README.md]: Moby Dick: Use Docker to run the app
Moby Dick II: Use Docker Compose to run the app,

I feel ripped off, that was not announced.

File: description.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
 # Description of the contents of this project
 
-Complete me!
+## No Docker Usage
+
+## Docker Usage
+
+to build the docker image and run the container:
+```sh=
```
Comment (fjlopez) [inline description.md:8]: Remove the `=` and add an empty line before

File: description.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+# Build image
+docker build . -t lab1-git-race -f .\src\main\docker\app.dockerfile
+
+# Run container
+docker run -d -p 8080:8080 --name lab1-git-race-container lab1-git-race
+# To stop the container use: docker stop <identifier:hash>
+```
+
+## Docker-compose
+```
```
[the remark this answers is not in the document]
Comment (fjlopez) [reply description.md:21]: Remember, add the language and add an empty line before all the code blocks

**Review by fjlopez — CHANGES_REQUESTED** [review]

**Review by hec7orci7o — COMMENTED** [review]
now should be fine

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fjlopez) [reply README.md]: Check the slides in moodle on "¿Como premiar el trabajo individual?" and also in the GUIDE.md

image

**Review by hec7orci7o — COMMENTED** [review]
updated files that differ locally to avoid merge conflicts

**Review by fjlopez — APPROVED** [review]

**Pull request merged**